sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientEncryptionKey.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosClientEncryptionKeyResponse;
+import com.azure.cosmos.util.Beta;
+
+/**
+ * The type Cosmos clientEncryptionKey. This contains methods to operate on a cosmos clientEncryptionKey
+ * synchronously
+ */
+@Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public class CosmosClientEncryptionKey {
+    private final CosmosDatabase database;
+    private String id;
+    private final CosmosAsyncClientEncryptionKey cosmosAsyncClientEncryptionKey;
+
+    CosmosClientEncryptionKey(String id,
+                              CosmosDatabase database,
+                              CosmosAsyncClientEncryptionKey cosmosAsyncClientEncryptionKey) {
+        this.database = database;
+        this.id = id;
+        this.cosmosAsyncClientEncryptionKey = cosmosAsyncClientEncryptionKey;
+    }
+
+    /**
+     * Get the id of the {@link CosmosClientEncryptionKey}
+     *
+     * @return the id of the {@link CosmosClientEncryptionKey}
+     */
+    @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
Same here, and other places, we need to update this to `V4_16_0`

Reply:
done